crates/contract/tests/snapshots/abi__abi_has_not_changed.snap

```diff
@@ -2798,6 +2798,72 @@ expression: abi
           "format": "uint64",
           "minimum": 0.0
         },
+        "VerifiedAttestation": {
+          "oneOf": [
+            {
+              "type": "object",
+              "required": [
+                "Dtack"
+              ],
+              "properties": {
+                "Dtack": {
+                  "$ref": "#/definitions/VerifiedDstackAttestation"
+                }
+              },
+              "additionalProperties": false
+            },
+            {
+              "type": "object",
+              "required": [
+                "Mock"
+              ],
+              "properties": {
+                "Mock": {
+                  "$ref": "#/definitions/MockAttestation"
+                }
+              },
+              "additionalProperties": false
+            }
+          ]
+        },
+        "VerifiedDstackAttestation": {
+          "type": "object",
+          "required": [
+            "expiry_timestamp_seconds",
+            "launcher_compose_hash",
+            "mpc_image_hash"
+          ],
+          "properties": {
+            "expiry_timestamp_seconds": {
+              "description": "Unix time stamp for when this attestation will be expired.",
+              "type": "integer",
+              "format": "uint64",
+              "minimum": 0.0
+            },
+            "launcher_compose_hash": {
+              "description": "The digest of the launcher compose file running.",
+              "type": "array",
+              "items": {
+                "type": "integer",
+                "format": "uint8",
+                "minimum": 0.0
+              },
+              "maxItems": 32,
+              "minItems": 32
+            },
+            "mpc_image_hash": {
+              "description": "The digest of the MPC image running.",
+              "type": "array",
+              "items": {
+                "type": "integer",
+                "format": "uint8",
+                "minimum": 0.0
+              },
+              "maxItems": 32,
+              "minItems": 32
+            }
+          }
+        },
```

Comment:
Given that we change the interface here, I'm curious about compatibility guarantees. We've discussed this synchronously now and it seems like this would cause nodes to fail to observe attestations in previous contract versions.

We need to figure out how to make this change backwards compatible to ensure nodes can submit and observe attestations in the old contract.